plugin.video.eitb/addon.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.eitb" name="EITB Nahieran (unofficial)" version="3.0.0" provider-name="Mikel Larreategi">
+    <requires>
+        <import addon="xbmc.python" version="3.0.0"/>
+        <import addon="script.module.requests" version="2.22.0"/>
+    </requires>
+    <extension point="xbmc.python.pluginsource" library="addon.py">
+        <provides>video audio</provides>
+    </extension>
+    <extension point="xbmc.addon.metadata">
+        <platform>all</platform>
+        <summary lang="en_GB">Plugin for EITB online service</summary>
+        <description lang="en_GB">Plugin for EITB using the unofficial REST API</description>
+        <summary lang="eu">EITB Nahieranentzako plugina</summary>
+        <description lang="eu">EITB Nahieranen API ez ofiziala erabiliz egindakoa</description>
+        <summary lang="es">Plugin para EITB a la carta</summary>
```

Comment:
The previous version worked perfectly using only 'es', just the language without country signal...

Reply:
and? Kodi v14 and older uses ISO-639 code, which have been updated since then (it works due to backwards compatibility but no guarantees it will work forever).

See: https://kodi.wiki/view/Addon.xml

Please update as advised

Reply:
changed to be es_ES